index.js

```diff
@@ -16,15 +12,24 @@ async function main() {
         await exec.exec('pip', ['freeze', '--local']);
     });
 
+    const extra_args = core.getInput('extra_args');
+    if (!Array.isArray(extra_args)) {
+        throw new Error('`extra_args` must be an `Array`');
+    } else {
```

Comment:
is it needed to type check this? what happens in the case where we just let it through (does `...` raise a reasonable error message?)

Reply:
I think we do. If we just let it through:

![image](https://user-images.githubusercontent.com/14007150/81776263-24c2ed00-94b4-11ea-88bf-8ad611d00de3.png)

Not the most reasonable error message I've seen. :joy:

FYI my configuration looks like:

```yaml
    - uses: deepyaman/action@v1.0.3
      with:
        extra_args: --files ${{ env.FILES }}
        token: ${{ secrets.GITHUB_TOKEN }}
```

where `env.FILES` is the list of changed files (in this case, just `.github/workflows/actions.yaml`).

If we type check it:

![image](https://user-images.githubusercontent.com/14007150/81777331-373e2600-94b6-11ea-9b9a-8e36f48456d6.png)

Reply:
sounds good!  that error is pretty funny though you gotta admit 🤣

Reply:
Actually... I was testing, and remembered why I used a string:

![image](https://user-images.githubusercontent.com/14007150/81777925-7c168c80-94b7-11ea-8bb5-2451ad9a615d.png)

Number of posts on the topic (e.g. https://github.community/t5/GitHub-Actions/Can-action-inputs-be-arrays/td-p/33776).

Let me switch it back to parsing a string and give you something to poke at.